packages/location-management/src/components/LocationForm/index.tsx

```diff
@@ -63,6 +64,7 @@ export interface LocationFormProps
   onCancel: () => void;
   service: typeof OpenSRPService;
   username: string;
+  dispatch: Dispatch<FetchedTreeAction>;
```

Comment:
@ashfahan Why are we not using `useDispatch` from the component and passing dispatch as prop?

Reply:
this was something that was done by @p-netm that why i didn't changed the implementation but yes as far as i am aware of changing the implementation won't any impact on implementation